GitCommands/Git/GitItemStatus.cs

```diff
@@ -146,6 +149,16 @@ public bool IsStatusOnly
             set => SetFlag(value, Flags.IsStatusOnly);
         }
 
+        /// <summary>
+        /// This item is not a native Git item, just status information
+        /// The item is to be calculated with Git range-diff
+        /// </summary>
```

Comment:
```suggestion
        /// <remarks>
        /// This item is not a native git item, but a status information
        /// calculated with git range-diff command.
        /// </remarks>
```

Reply:
Changed to <remarks> for IsStatusOnly too